ITEventsTests/SearchParametersServiceTests.swift

```diff
@@ -0,0 +1,30 @@
+import XCTest
+@testable import ITEvents
+
+class SearchParametersServiceTests: XCTestCase {
+    var searchParametersService: SearchParametersService!
```

Comment:
All your code in tests should also work with a protocol not with implementation:
```swfit
var searchParametersService: ISearchParametersService!
```

And you will see why fixing this test